lib/actions/ui.js

```diff
@@ -212,12 +212,33 @@ export function moveTo(i) {
 }
 
 export function showPreferences() {
+  const isWin = process.platform === 'win32';
+  // eslint-disable-next-line no-template-curly-in-string
+  const command = isWin ? ' start notepad "%userprofile%\\.hyper.js"' : ' bash -c \'exec env ${EDITOR:=nano} ~/.hyper.js\'';
+  const message = [];
+  message.push(isWin ?
+    ' echo Attempting to open ^%userprofile^%\\.hyper.js with notepad' :
+    ' echo Attempting to open ~/.hyper.js with your \\$EDITOR');
+  message.push(' echo If it fails, open it manually with your favorite editor!');
+
   return dispatch => {
     dispatch({
       type: UI_SHOW_PREFERENCES,
       effect() {
-        dispatch(
-          shell.showItemInFolder(path.join(os.homedir(), '.hyper.js')));
+        if (isWin) {
```

Comment:
If this does go further then, you'd be able to remove all the stuff you re-added at the top of this function having to do with the message & command on Windows. I'll put a note on it.